tests/integration/test_user_report.py

```diff
@@ -0,0 +1,64 @@
+import json
+import uuid
+
+from sentry_sdk.envelope import Envelope, Item, PayloadRef
+
+
+def test_standalone_user_report(
+    relay_with_processing, mini_sentry, attachments_consumer
+):
+    project_id = 42
+    relay = relay_with_processing()
+    mini_sentry.add_full_project_config(project_id)
+
+    report_payload = {
+        "name": "Josh",
+        "email": "",
+        "comments": "I'm having fun",
+        "event_id": "4cec9f3e1f214073b816e0f4de5f59b1",
+    }
+
+    relay.send_user_report(
+        project_id,
+        report_payload,
+    )
+
+    attachments_consumer = attachments_consumer()
+    report = attachments_consumer.get_user_report(timeout=5)
+    assert json.loads(report["payload"]) == report_payload
+
+
+def test_user_report_with_event(
+    relay_with_processing, mini_sentry, attachments_consumer
+):
+    project_id = 42
+    relay = relay_with_processing()
+    mini_sentry.add_full_project_config(project_id)
+
+    event_id = uuid.uuid1().hex
+
+    error_payload = {
```

Comment:
I think we could benefit from a test where the error is dropped but not the user report, what do you think?

Reply:
That's might be good, what condition do you have in mind for this to happen?

Reply:
That's might be good, what condition do you have in mind for this to happen?

Reply:
Sending an incorrect error should be good enough; but alternatively, send an error that's filtered out by e.g. inbound filters. The user report should be consumed in both cases.